### ISSUE TITLE
MC photo electron image is zero for all pixels and telescopes

### ISSUE THREAD

**Issue by kpfrang** [cta-observatory/pyhessio#73]
When looking at the mc information read by `HESSIOEventSource` I realized that `photo_electron_image` is an array of zeros for all telescopes:

```python
source = EventSourceFactory.produce(input_url=input_file,
                                    max_events=2)
for event in source:
    for tel_id in event.r0.tels_with_data:
        print("Sum of photo_electron_image: ", sum(event.mc.tel[tel_id].photo_electron_image))
```
results in `Sum of photo_electron_image:  0` for all telescopes and events. Am I missing something here or is there a bug in [get_mc_number_photon_electron](https://github.com/cta-observatory/pyhessio/blob/7b74e5faaa4e15a07219f06806574b22cbf096ee/pyhessio/src/pyhessio.c#L534)?

I'm using pyhessio 2.1.1

**Comment by maxnoe** [comment]
Can you provide your input data? The files prod3b files I checked simply do not contain this information, so hessio sets it to 0.

**Comment by kpfrang** [comment]
Thanks for checking Max. For each of point like gammas, diffuse gammas, electrons and protons I tried with several prod3b files and it always gave 0. 

So not all of the simulations do have this information stored? And if that's true, which files do have the photo electron image stored?

**Comment by maxnoe** [comment]
you could check if you do `pip install eventio`

and run 
```
$ eventio_print_object_information /path/to/file
```

And check if there are containers of type 1208

**Comment by kpfrang** [comment]
Yes you are right, the files I looked at didn't have this container. 

Unfortunately, I did not find simulations with this information. Don't we have such files?
